src/app/bulk-edit/bulk-edit-editor/bulk-edit-editor.component.ts

```diff
@@ -208,10 +226,28 @@ export class BulkEditEditorComponent implements OnInit {
   }
 
   private screenResize() {
+    const nonStandardFields = this.nonStandardColWidths.map((c) => c.field);
+
     const columnIds = this.gridColumnApi
       .getAllColumns()
+      .filter((col) => !nonStandardFields.includes(col.getColDef().field))
       .map((col) => col.getColId());
+
     this.gridColumnApi.autoSizeColumns(columnIds);
+
+    this.nonStandardColWidths
```

Comment:
I've got a method for non standard col resizable, if we need more of these may be worth structuring as something we can dynamically adjust via properties or something?